soccer/gameplay/skills/capture.py

```diff
@@ -99,7 +99,8 @@ def on_enter_course_approach(self):
 
     def execute_course_approach(self):
         # don't hit the ball on accident
-        self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        #self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        self.robot.disable_avoid_ball()
```

Comment:
Well, the problem is that the path around the ball is usually pretty crappy because the robot can usually either catch up the the ball and capture it or the balls going at high speed, its usually going to go out.